lib/tasks_private/spec.rake

```diff
@@ -7,5 +7,5 @@ desc "Run all specs"
 RSpec::Core::RakeTask.new(:spec => ["app:test:initialize", "app:evm:compile_sti_loader"]) do |t|
   spec_dir = File.expand_path("../../spec", __dir__)
   EvmTestHelper.init_rspec_task(t, ['--require', File.join(spec_dir, 'spec_helper')])
-  t.pattern = FileList[spec_dir + '/**/*_spec.rb'].exclude(spec_dir + '/manageiq/**/*_spec.rb')
+  t.pattern = FileList[spec_dir + '/**/*_spec.rb']
```

Comment:
@cfcosta Please see the changes in https://github.com/ManageIQ/manageiq-automation_engine/pull/23